src/profiles.js

```diff
@@ -0,0 +1,270 @@
+export default {
```

Comment:
This is awesome.

I'd like to add a step to my build process that can generate this file from the vendored `quiet/quiet-profiles.json` that my PR is adding. I haven't quite decided how I want to template it, but that would help keep this in sync with that file.